runtime/include/chpl-wide-ptr-fns.h

```diff
@@ -37,7 +37,7 @@
 // chpl_error needed for some of the packed wide pointer error cases.
 #include "error.h"
 
-#define WIDE_PTR_INLINE ___always_inline
+#define WIDE_PTR_INLINE inline
```

Comment:
I don't see any benefit of this WIDE_PTR_INLINE macro, but I do see a bit of obfuscation. Would you mind getting rid of it and changing all the uses to plain "inline" while you're here?